#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_fix_manpage.dpatch by Sandro Tosi <matrixhasu@gmail.com>
##
## DP: Added to escape minus signs in manpage (lintian warning)

@DPATCH@
diff -urNad fdupes~/fdupes.1 fdupes/fdupes.1
--- fdupes~/fdupes.1	2008-03-17 20:12:35.000000000 +0100
+++ fdupes/fdupes.1	2008-03-18 23:30:49.051348038 +0100
@@ -53,7 +53,7 @@
 below)
 .TP
 .B -N --noprompt
-when used together with --delete, preserve the first file in each
+when used together with \-\-delete, preserve the first file in each
 set of duplicates and delete the others without prompting the user 
 .TP
 .B -v --version
